docker/docker-compose.yml

```diff
@@ -40,7 +41,7 @@ services:
 
   data-consumer:
     container_name: data-consumer
-    build: https://github.com/MetPX/sarracenia.git#v03_wip
+    image: webbben/sarracenia
```

Comment:
Same here, shouldn't we point to the canonical repo?

Reply:
Same as ^